xwiki-platform-core/xwiki-platform-test/xwiki-platform-test-ui/src/main/java/org/xwiki/test/ui/po/CommentsTab.java

```diff
@@ -68,7 +85,7 @@ public void setAnonymousCommentAuthor(String author)
 
     public int getCommentID(String content)
     {
-        this.commentsList = getDriver().findElementsWithoutWaiting(By.className("xwikicomment"));
+        this.commentsList = getDriver().findElements(By.className("xwikicomment"));
```

Comment:
Why changing that?

Reply:
At some point not waiting was making the test flaky but apparently it is not the case anymore, reverting